Update borders.md (#2799)
enkidevs/curriculum: web/styling/box-model/borders.md

## Content

Borders occupy the space between margin and padding, providing an outline of the element.

> The `border` has 3 properties: `width`, `style`, and `color`. Each of them can be declared individually:

Reviewer: hmm, let's leave out the "the" at the start; we could say "The `border` property" but then we'd repeat ourselves "The `border` property has three properties"

```suggestion
`border` has three properties: `width`, `style`, and `color`. Each of them can be declared individually:
```

}
```

> Or using the shorthand notation:

Reviewer: what do you think about this?
```suggestion
Or you can use the shorthand notation:
```

```css
div{
  border: red 4px solid;
}
```

> You can declare it with any combination of the `width`, `style`, and `color`, as well as in any order.

Reviewer: what do you think about this? I tried to highlight that the order and combination doesn't matter
```suggestion
You can declare it with **any combination** of `width`, `style`, and `color` in **any order**.
```

Author: yeah  it is better

> > ❗ The `style` needs to be set for the border to show. It is because the default value is `none`. 

Reviewer: let's switch to the active voice

```suggestion
> ❗ You need to set the `style` for the border to show. If you don't, the default value is `none`. 
```

Author: I don't know why, I always go for passive 😄

> Borders can have different styles, of which the most common is `solid`. Others are `dashed`, `dotted`, or `double`.

Reviewer: imo we can make this one phrase

if there are no other styles, remove the ", and others" at the end of the phrase

```suggestion
Borders can have different styles such as `solid` (**the most common**), `dashed`, `dotted`, `double`, and others.
```

Author: Yeah, there are a few 
border-style: none;
border-style: hidden;
border-style: dotted;
border-style: dashed;
border-style: solid;
border-style: double;
border-style: groove;
border-style: ridge;
border-style: inset;
border-style: outset;
